libr/core/core.c

```diff
@@ -1029,6 +1034,97 @@ static const char *radare_argv[] = {
 	NULL
 };
 
+static void autocomplete_ms_path(RLineCompletion *completion, RCore *core, const char *str, const char *path) {
+	char *lpath = NULL, *dirname = NULL , *basename = NULL;
+	char *home = NULL, *p = NULL;
+  	char *pwd = *(core->rfs->cwd);
+	int n = 0;
+	RList *list;
+	RListIter *iter;
+	RFSFile *file;
+
+	if (!path) {
+		goto out;
+	}
+
+	lpath = r_str_new (path);
+	p = (char *)r_str_last (lpath, R_SYS_DIR);
+	if (p) {
+		*p = 0;
+		if (p == lpath) { // /xxx
+			dirname  = r_str_new ("/");
+		} else if (lpath[0] == '~' && lpath[1]) { // ~/xxx/yyy
```

Comment:
i dont think r_str_home is suposed to work on remote mountpoint. i would skip that

Reply:
Removed the r_str_home logic